.github/workflows/e2e.yaml

```diff
@@ -114,7 +114,8 @@ jobs:
           sed -i -e "s|nameOverride:.*|nameOverride: kyverno|g" charts/kyverno/values.yaml
           sed -i -e "s|fullnameOverride:.*|fullnameOverride: kyverno|g" charts/kyverno/values.yaml
           sed -i -e "s|namespace:.*|namespace: kyverno|g" charts/kyverno/values.yaml
-          sed -i -e "s|tag:  # replaced in e2e tests|tag: $(git describe --match "v[0-9]*")|" charts/kyverno/values.yaml
+          sed -i -e "s|tag:.*|tag: $(git describe --tag)|" charts/kyverno/values.yaml
```

Comment:
@treydock - this command also replaces tag for busybox container, is that what we want?

Reply:
I'm not sure I know what busybox container you're referring to?  When I run the command it replaces tag for main Kyverno container and init container:

```
$ sed -i -e "s|tag:.*|tag: $(git describe --tag)|" charts/kyverno/values.yaml
$ git diff
diff --git a/charts/kyverno/values.yaml b/charts/kyverno/values.yaml
index 696df1a4..63f8e94f 100644
--- a/charts/kyverno/values.yaml
+++ b/charts/kyverno/values.yaml
@@ -16,14 +16,14 @@ rbac:
 image:
   repository: ghcr.io/kyverno/kyverno
   # Defaults to appVersion in Chart.yaml if omitted
-  tag:  # replaced in e2e tests
+  tag: 1.6-dev-127-g79006ec2
   pullPolicy: IfNotPresent
   pullSecrets: []
   # - secretName
 initImage:
   repository: ghcr.io/kyverno/kyvernopre
   # If initImage.tag is missing, defaults to image.tag
-  tag:  # replaced in e2e tests
+  tag: 1.6-dev-127-g79006ec2
   # If initImage.pullPolicy is missing, defaults to image.pullPolicy
   pullPolicy:
   # No pull secrets just for initImage; just add to image.pullSecrets
```